xbmc/Util.cpp

```diff
@@ -1563,9 +1561,8 @@ void CUtil::GetSkinThemes(std::vector<std::string>& vecTheme)
   CFileItemList items;
   CDirectory::GetDirectory(strPath, items, "", DIR_FLAG_DEFAULTS);
   // Search for Themes in the Current skin!
-  for (int i = 0; i < items.Size(); ++i)
+  for (auto pItem : items)
```

Comment:
missing `const` modifier looks wrong to me

Reply:
also reference - `for (const auto &item : items)`

Reply:
Look at the old code. It also created a local non-`const` copy of every item, and I preserved that:
```
-  for (int i = 0; i < items.Size(); ++i)
+  for (auto pItem : items)
   {
-    CFileItemPtr pItem = items[i];
```
Of course, I can change that, but IMHO that is out of this commit's scope, because that commit was supposed to refactor a loop to a different and easier syntax, and nothing else.

Reply:
Please change to `for (const auto pItem& : items)` if no additional const changes (called function in that block are needed).
If you don't do it, it won't be done.